.github/workflows/sca-scan.yml

```diff
@@ -7,7 +7,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - run: npx lerna bootstrap --force-local --concurrency 1
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install
+      # - run: npx lerna bootstrap --force-local --concurrency 1
       - name: Run Snyk to check for vulnerabilities
         uses: snyk/actions/node@master
```

Comment:
@antonyagustine please checkout the release workflow, if it effects this change, we need to update that as well

Reply:
@shafeeqd959, @netrajpatel please ignore this PR don't spend any more time reviewing it. It is created for the POC purpose, It's still in the exploring stage.

Reply:
@shafeeqd959 It won't affect any release workflow.

Thanks,